graphql/queryResolvers/setStar.ts

```diff
@@ -0,0 +1,44 @@
+import db from '../../helpers/dbload'
+import { LoggedRequest } from '../../@types/helpers'
+import { Star as StarType } from '../../@types/lesson'
+import _ from 'lodash'
+
+const { Star } = db
+
+export const setStar = async (
+  _parent: void,
+  arg: StarType,
+  ctx: { req: LoggedRequest }
+) => {
+  const { req } = ctx
+  try {
+    const studentId = _.get(req, 'user.id', false)
+    if (!studentId) {
+      throw new Error('Student is not logged in')
+    }
+
+    const { mentorId, lessonId } = arg
+    if (!mentorId) {
+      throw new Error('Missing or invalid mentorId')
+    }
+    if (!lessonId) {
+      throw new Error('Missing or invalid lessonId')
```

Comment:
Do you need to check for `mentorId` and `lessonId`? Can't you just mark them as required in typedef?

Reply:
Right -- I don't need to check if they are missing, I will update my tests and error description accordingly. However, I do need to check if the inputs are valid (`i.e someone can't pass in the number 0`)